src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,6 @@
 *REMOVED*override abstract Microsoft.CodeAnalysis.CompilationOptions.GetHashCode() -> int
 *REMOVED*static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromMetadata(System.IntPtr metadata, int size, System.IDisposable! owner, bool disposeOwner) -> Microsoft.CodeAnalysis.ModuleMetadata!
+abstract Microsoft.CodeAnalysis.SemanticModel.GetPreprocessingSymbolInfoCore(Microsoft.CodeAnalysis.SyntaxToken token) -> Microsoft.CodeAnalysis.PreprocessingSymbolInfo
```

Comment:
You'll need to go through API review process for proposed public API addition.
[https://github.com/dotnet/roslyn/blob/main/docs/contributing/API%20Review%20Process.md](https://github.com/dotnet/roslyn/blob/main/docs/contributing/API%20Review%20Process.md)

Tagging @333fred as FYI

Reply:
There's even more coming, will push my progress tomorrow

Reply:
@Rekkonnect please follow our public API process _before_ proposing PRs, as we documented in the process: https://github.com/dotnet/roslyn/blob/main/docs/contributing/API%20Review%20Process.md#pull-requests.